pettingzoo/butterfly/cooperative_pong/cooperative_pong.py

```diff
@@ -10,8 +10,10 @@
 from pettingzoo.utils.agent_selector import agent_selector
 from pettingzoo.utils.conversions import parallel_wrapper_fn
 
+from .ball import Ball
 from .cake_paddle import RENDER_RATIO, CakePaddle
```

Comment:
Moving `RENDER_RATIO` to `cooperative_pong.py` causes a circular dependency issue, but I agree that this should be a parameter of the environment as a whole rather than being contained in the cake paddle class. I've moved it to `cooperative_pong.py`, and I've also moved the global variables to keyword arguments of the environment.